operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,242 @@
+# iSCSI SBPS (Scalable Boot Content Projection Service)
+
+iSCSI based boot content projection solution named Scalable Boot Content Projection Service (SBPS)

```

Comment:
This whole file has `\r\n` line terminators.  Please strip out all the `\r` characters.

Reply:
You can remove the `\r` characters with:
```bash
cat iscsi_sbps.md | tr -d '\r' > x
mv x iscsi_sbps.md
```